src/EnergyPlus/UnitarySystem.cc

```diff
@@ -10260,11 +10260,7 @@ namespace UnitarySystems {
                 DataHVACGlobals::MSHPMassFlowRateLow = state.dataUnitarySystems->CompOnMassFlow * PartLoadRatio; // proportional to PLR when speed = 1,  #5518
             }
         } else if (this->m_DiscreteSpeedCoolingCoil && (state.dataUnitarySystems->CoolingLoad && CoolSpeedNum == 1)) {
```

Comment:
@lgu1234 I tested with a multispeed air source heat pump test file, it seems that cooling curve inputs are fixed as expected, now airflow curve inputs for both speeds are constant, good job. However, I noticed that the heating curve inputs are showing the same issue at low speed. Can you look into heating as well?